clients/src/test/java/org/apache/kafka/common/security/authenticator/SaslAuthenticatorFailureDelayTest.java

```diff
@@ -53,7 +53,7 @@
 @RunWith(value = Parameterized.class)
 public class SaslAuthenticatorFailureDelayTest {
     private static final int BUFFER_SIZE = 4 * 1024;
-    private static MockTime time = new MockTime(50);
+    private static MockTime time = new MockTime(10);
```

Comment:
Any reason why this isn't final?

Reply:
Made it final and changed to instance variable like we have in other tests.